using/src/schema.rs

```diff
@@ -0,0 +1,693 @@
+//! A GraphQL schema referencing one or more specifications
```

Comment:
I would either move this to right above `pub struct Schema<'a>`, so that the docs built directly for the `Schema` struct, or, add another set of comments just above there, since this page is currently empty:
<img width="1036" alt="Screen Shot 2021-01-19 at 19 17 30" src="https://user-images.githubusercontent.com/8107784/105076165-01146580-5a8b-11eb-8858-6f9dca31d52d.png">